### PR TITLE
Replace all environment variables instead of only API_URL

### PR DESCRIPTION
**Pull request by yasinuslu**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [x] 🛠 Workflow CI/CD changes

## Related Issues or Documents

Instead of replacing only the API_URL, docker image now replaces all environment variables available to the index.html

## Have you written tests?
- [ ] Yes!
- [x] No... here is why: This is an update to the `replace-api-url.sh` it didn't have tests before, so I didn't add it. We can figure out a way to write a test for it if you request it.

### CODE REVIEW

**Review by lkostrowski — COMMENTED** [review]
I think the improvement is great. 

@krzysztofzuraw can you verify this against your latest work on dockerization?

**Review by NyanKiyoshi — APPROVED** [review]

**Review by poulch — APPROVED** [review]

File: nginx/replace-api-url.sh
Comment (krzysztofzuraw) [inline nginx/replace-api-url.sh:1]: Good job - works as it should - can you change the filename to `replace-env-vars` instead of `replace-api-url.sh`?

**Review by poulch — APPROVED** [review]

**Review by NyanKiyoshi — DISMISSED** [review]

File: nginx/replace-api-url.sh
Comment (yasinuslu) [reply nginx/replace-api-url.sh:1]: of course

**Review by poulch — APPROVED** [review]

**Review by NyanKiyoshi — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Comment by krzysztofzuraw** [comment]
@yasinuslu thank you for your work 🙇🏻‍♂️

**Pull request merged**